best-practices/styling/index.html

```diff
@@ -885,23 +885,49 @@ <h3>Alignment, positioning and spatial arrangement</h3>
 			<section id="text-spacing">
 				<h3>Text spacing</h3>
 
-				<div class="issue" title="Seeking an example of increased line spacing"><span/></div>
-
-				<div class="issue" title="Seeking an example of increased inter-word spacing"><span/></div>
-
+				<div class="issue" title="Need HTML and CSS for increased line spacing"><span/></div>
+				<aside class="example" title="Paragraph and heading that are double spaced">
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠁⠇⠊⠉⠑⠀⠊⠝⠀⠠⠺⠕⠝⠙⠑⠗⠇⠁⠝⠙⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠨⠶⠠⠁⠇⠊⠉⠑⠀⠊⠝⠀⠠⠺⠕⠝⠙⠑⠗⠇⠁⠝⠙⠨⠄⠀⠊⠎⠀⠁⠀⠎⠞⠕⠗⠽⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠃⠕⠥⠞⠀⠁⠀⠛⠊⠗⠇⠀⠝⠁⠍⠑⠙⠀⠠⠁⠇⠊⠉⠑⠀⠺⠓⠕⠀⠋⠁⠇⠇⠎⠀⠙⠕⠺⠝
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠀⠗⠁⠃⠃⠊⠞⠀⠓⠕⠇⠑⠀⠁⠝⠙⠀⠑⠝⠞⠑⠗⠎⠀⠁⠀⠍⠁⠛⠊⠉⠁⠇⠀⠺⠕⠗⠇⠙
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠋⠥⠇⠇⠀⠕⠋⠀⠎⠞⠗⠁⠝⠛⠑⠀⠁⠝⠙⠀⠺⠕⠝⠙⠑⠗⠋⠥⠇⠀⠉⠗⠑⠁⠞⠥⠗⠑⠎⠲
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠁⠎⠀⠎⠓⠑⠀⠑⠭⠏⠇⠕⠗⠑⠎⠀⠞⠓⠊⠎⠀⠝⠑⠺⠀⠏⠇⠁⠉⠑⠂⠀⠎⠓⠑⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠍⠑⠑⠞⠎⠀⠞⠁⠇⠅⠊⠝⠛⠀⠁⠝⠊⠍⠁⠇⠎⠂⠀⠁⠀⠛⠗⠥⠍⠏⠽⠀⠟⠥⠑⠑⠝⠂⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠝⠙⠀⠑⠧⠑⠝⠀⠁⠀⠙⠊⠎⠁⠏⠏⠑⠁⠗⠊⠝⠛⠀⠉⠁⠞⠲⠀⠠⠁⠇⠊⠉⠑⠀⠓⠁⠎⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠞⠕⠀⠥⠎⠑⠀⠓⠑⠗⠀⠊⠍⠁⠛⠊⠝⠁⠞⠊⠕⠝⠀⠁⠝⠙⠀⠉⠥⠗⠊⠕⠎⠊⠞⠽⠀⠞⠕⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠋⠊⠝⠙⠀⠓⠑⠗⠀⠺⠁⠽⠀⠞⠓⠗⠕⠥⠛⠓⠀⠁⠇⠇⠀⠞⠓⠑⠀⠉⠗⠁⠵⠽⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠙⠧⠑⠝⠞⠥⠗⠑⠎⠀⠊⠝⠀⠠⠺⠕⠝⠙⠑⠗⠇⠁⠝⠙⠲⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀</code></pre>
+				</aside>
+				<div class="issue" title="Need HTML and CSS for increased inter-word spacing"><span/></div>
+				<aside class="example" title="Line numbered text with inter-word spacing"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠠⠽⠕⠥⠀⠺⠊⠇⠇⠀⠗⠑⠚⠕⠊⠉⠑⠀⠞⠕⠀⠓⠑⠁⠗⠀⠞⠓⠁⠞⠀⠝⠕⠀⠀⠀⠀⠀⠼⠉
+⠙⠊⠎⠁⠎⠞⠑⠗⠀⠓⠁⠎⠀⠁⠉⠉⠕⠍⠏⠁⠝⠊⠑⠙⠀⠞⠓⠑⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠉⠕⠍⠍⠑⠝⠉⠑⠍⠑⠝⠞⠀⠕⠋⠀⠁⠝⠀⠑⠝⠞⠑⠗⠏⠗⠊⠎⠑⠀⠺⠓⠊⠉⠓⠀⠀⠀⠼⠙
+⠽⠕⠥⠀⠓⠁⠧⠑⠀⠗⠑⠛⠁⠗⠙⠑⠙⠀⠺⠊⠞⠓⠀⠀⠀⠎⠥⠉⠓⠀⠑⠧⠊⠇⠀⠀⠀⠀⠼⠑
+⠋⠕⠗⠑⠃⠕⠙⠊⠝⠛⠎⠲⠀⠠⠊⠀⠁⠗⠗⠊⠧⠑⠙⠀⠓⠑⠗⠑⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠽⠑⠎⠞⠑⠗⠙⠁⠽⠂⠀⠁⠝⠙⠀⠍⠽⠀⠋⠊⠗⠎⠞⠀⠞⠁⠎⠅⠀⠀⠀⠊⠎⠀⠞⠕⠀⠀⠼⠋
+⠁⠎⠎⠥⠗⠑⠀⠍⠽⠀⠙⠑⠁⠗⠀⠎⠊⠎⠞⠑⠗⠀⠕⠋⠀⠍⠽⠀⠺⠑⠇⠋⠁⠗⠑⠀⠀⠀⠀⠀
+⠁⠝⠙⠀⠊⠝⠉⠗⠑⠁⠎⠊⠝⠛⠀⠉⠕⠝⠋⠊⠙⠑⠝⠉⠑⠀⠀⠀⠊⠝⠀⠞⠓⠑⠀⠀⠀⠀⠼⠛
+⠎⠥⠉⠉⠑⠎⠎⠀⠕⠋⠀⠍⠽⠀⠥⠝⠙⠑⠗⠞⠁⠅⠊⠝⠛⠲⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀</code></pre>
+				</aside>
```

Comment:
This example of line numbered text is unfortunately not really an example of inter-word spacing, at least not the way I meant it.

With inter-word spacing I meant when _all_ words in a piece of text are separated by multiple spaces (to increase the readability for beginner braille readers). In this example, you could say that the increased spacing at specific places are "marks", or "gaps" where the numbers would be if they wouldn't be floated to the right.

We have covered this example already in the "[Line numbered prose](https://daisy.github.io/ebraille/best-practices/styling/#example-line-numbered-prose) under the section "Floating content" and I think it kind of fits well with the floating aspect. We could possibly use the example here too, but then I would omit the line numbers (and refer to the other example), and then I would still like an additional example where I can use the [`word-spacing`](https://drafts.csswg.org/css-text/#word-spacing-property) property.

Reply:
Thanks @bertfrees . After reviewing `word-spacing`, I'm not sure that it is generally relevant to braille. The closest I can think of is on a device where equidistant pins are utilized, it would be helpful to be told when the spatial arrangement is important so that we can ensure to respect the space between words under all circumstances. With equidistant pins it would make sense to sometimes limit the space between braille cells if it would lead to a large gap (because of the equidistant spacing), like when you have a left-handed cell that is followed by a space and then a right-handed cell. Making the space somewhat smaller does make the space look more like it would on paper but it also affects the layout of the braille so that one line will not line up with the following line like it maybe should. Would this very specific example qualify? If so, something like spatial math would satisfy this requirement.

Reply:
The use case I was thinking of is when it is used in combination with `letter-spacing`, to create blank cells between the braille cells (for unexperienced braille readers). 

When letter-spacing is 1ch, word-spacing could be 2ch or 3ch.

```
W O R D  W O R D
```

or

```
W O R D   W O R D
```

This is a feature that I implemented in Pipeline for Celia (Finland), so it might be relevant for them at least.

Your use case of equidistant pins is also interesting. However, like line numbered prose, I don't know if the `word-spacing` property would be the right way to deal with it. You will need to wrap the two words that should have a reduced word spacing in a span. I guess if an authoring tool could create this markup automatically, it might be fine.

```html
before <span class="reduced-word-spacing">word word</span> after
```

Reply:
The example is commented out for now.